examples/simple-example.rs

```diff
@@ -229,38 +213,25 @@ impl<F: FieldExt> NumericInstructions<F> for FieldChip<F> {
                 // but we can only rely on relative offsets inside this region. So we
                 // assign new cells inside the region and constrain them to have the
                 // same values as the inputs.
-                let lhs = region.assign_advice(
-                    || "lhs",
-                    config.advice[0],
-                    0,
-                    || a.value.ok_or(Error::Synthesis),
-                )?;
-                let rhs = region.assign_advice(
-                    || "rhs",
-                    config.advice[1],
-                    0,
-                    || b.value.ok_or(Error::Synthesis),
-                )?;
-                region.constrain_equal(a.cell, lhs)?;
-                region.constrain_equal(b.cell, rhs)?;
-
-                // Now we can assign the multiplication result into the output position.
-                let value = a.value.and_then(|a| b.value.map(|b| a * b));
-                let cell = region.assign_advice(
-                    || "lhs * rhs",
-                    config.advice[0],
-                    1,
-                    || value.ok_or(Error::Synthesis),
-                )?;
-
-                // Finally, we return a variable representing the output,
-                // to be used in another part of the circuit.
-                out = Some(Number { cell, value });
-                Ok(())
+                a.0.copy_advice(|| "lhs", &mut region, config.advice[0], 0)?;
+                b.0.copy_advice(|| "rhs", &mut region, config.advice[1], 0)?;
+
+                // Now we can assign the multiplication result, which is to be assigned
+                // into the output position.
```

Comment:
This comment was missed; change it to be like the others.

Reply:
To avoid another round of re-ACKs, I'll defer this to #412.